modules/mcuboot/CMakeLists.txt

```diff
@@ -6,6 +6,26 @@ if (CONFIG_MCUBOOT)
   if (CONFIG_NRF53_MULTI_IMAGE_UPDATE)
     set_shared(IMAGE ${IMAGE_NAME} PROPERTY NRF53_MULTI_IMAGE_UPDATE
       ${CONFIG_NRF53_MULTI_IMAGE_UPDATE})
+  if (NOT DEFINED CONFIG_BOOT_UPGRADE_ONLY)
+    # Added color defines due to this coming from a childe image and being out
+    # putted as raw unformatted text to increase visibility.
+    string(ASCII 27 Esc)
+    set(ColorReset "${Esc}[m")
+    set(BoldRed     "${Esc}[1;31m")
+    message(WARNING "${BoldRed}
```

Comment:
have you tested this formatting in windows ?

Reply:
Nope. Could potentially not work?